src/logic/wallets/getWeb3.ts

```diff
@@ -44,26 +46,59 @@ export const ETHEREUM_NETWORK_IDS = {
   5: ETHEREUM_NETWORK.GOERLI,
   // $FlowFixMe
   42: ETHEREUM_NETWORK.KOVAN,
+  // $FlowFixMe
+  246: ETHEREUM_NETWORK.ENERGY_WEB_CHAIN,
+  // $FlowFixMe
+  73799: ETHEREUM_NETWORK.VOLTA,
+}
+
+export enum ExplorerTypes {
+  Tx = 'tx',
+  Address = 'address',
 }
 
-export const getEtherScanLink = (type, value) => {
+export const getExplorerLink = (type: ExplorerTypes, value: string): string => {
   const network = getNetwork()
-  return `https://${
-    network.toLowerCase() === 'mainnet' ? '' : `${network.toLowerCase()}.`
-  }etherscan.io/${type}/${value}`
+
+  const getEtherScanLink = (network: ETHEREUM_NETWORK, type: ExplorerTypes, value: string) =>
```

Comment:
should we put this method outside of `getExplorerLink`? (I am not a js expert but I would expect that the function gets recreated everytime how it is now)

Reply:
Sure, good point.